base/strings/io.jl

```diff
@@ -83,10 +83,14 @@ Everything written to this I/O stream is returned as a string.
 or a `Pair` specifying a property and its value. `sizehint` suggests the capacity
 of the buffer (in bytes).
 
-The optional keyword argument `context` can be set to `:key=>value` pair
-or an `IO` or [`IOContext`](@ref) object whose attributes are used for the I/O
-stream passed to `f`.  The optional `sizehint` is a suggested size (in bytes)
-to allocate for the buffer used to write the string.
+The optional keyword argument `context` can be set to `:key=>value` pair, a
```

Comment:
Not your deed, but isn't it more correct to write
```suggestion
The optional keyword argument `context` can be set to a `:key=>value` pair, a
```
?